src/herbpy/action/blocks.py

```diff
@@ -0,0 +1,172 @@
+import logging, numpy, openravepy, prpy
+from prpy.action import ActionMethod
+from prpy.planning.base import PlanningError
+from prpy.util import ComputeEnabledAABB
+
+logger = logging.getLogger('herbpy')
+
+class NoTSRException(Exception):
+    pass
+
+def _GrabBlock(robot, blocks, table, manip=None, preshape=[1.7, 1.7, 0.2, 2.45],
+              **kw_args):
```

Comment:
`preshape` should default to `None` and get set to the default value inside the function.